lib/membrane_live_web/plugs/auth.ex

```diff
@@ -14,17 +14,21 @@ defmodule MembraneLiveWeb.Plugs.Auth do
   def init(default), do: default
 
   def call(conn, _default) do
-    with {@auth_header_key, "Bearer " <> token} <-
-           find_header_value(conn.req_headers, @auth_header_key),
+    with "Bearer " <> token <- find_bearer(conn.req_headers),
          {:ok, %{"user_id" => user_id}} <- Tokens.auth_decode(token) do
       assign(conn, :user_id, user_id)
     else
       err ->
-        FallbackController.call(conn, TokenErrorInfo.get_error_info(err))
+        conn
+        |> FallbackController.call(TokenErrorInfo.get_error_info(err))
         |> halt()
     end
   end
 
-  defp find_header_value(headers, key),
-    do: Enum.find(headers, fn elem -> match?({^key, _value}, elem) end)
+  defp find_bearer(headers) do
+    case Enum.find(headers, fn elem -> match?({@auth_header_key, _value}, elem) end) do
+      nil -> {:error, :no_jwt_in_header}
+      {_key, value} -> value
+    end
+  end
```

Comment:
Why did you use `match?` instead of standard pattern match? Rewrite it with the use of pipes.